diff --git a/resources/config.edn b/resources/config.edn
index 47281e8..3c704b2 100644
--- a/resources/config.edn
+++ b/resources/config.edn
@@ -34,7 +34,7 @@
                :adapter            "postgresql"
                :username           #env PG_USERNAME
                :password           #env PG_PASSWORD
-               :database-name      "logseq"
+               :database-name      "postgres"
                :server-name        #or [#env PG_HOST "localhost"]
                :port-number        #or [#env PG_PORT 5432]
                :register-mbeans    false
diff --git a/src/main/app/views/home.clj b/src/main/app/views/home.clj
index 103496a..1b7f186 100644
--- a/src/main/app/views/home.clj
+++ b/src/main/app/views/home.clj
@@ -42,6 +42,7 @@
        "minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no",
        :name "viewport"}]
      [:link {:type "text/css", :href css-href, :rel "stylesheet"}]
+     [:link {:type "text/css", :href "https://unpkg.com/@tabler/icons@latest/iconfont/tabler-icons.min.css", :rel "stylesheet"}]
      (when-not db-exists?
        [:style ".videoWrapper {
   position: relative;
